src/engraving/libmscore/edit.cpp

```diff
@@ -2801,6 +2801,116 @@ void Score::deleteMeasures(MeasureBase* mbStart, MeasureBase* mbEnd, bool preser
         }
     }
 
+    if (!preserveTies) {
+        // Reconnect or remove slurs that intersect with these deleted measures
+        Fraction sTick = mbStart->tick();
+        Fraction eTick = mbEnd->tick();
+        for (auto overlappingSlur : spannerMap().findOverlapping(sTick.ticks(), eTick.ticks())) {
+            Spanner* sp = overlappingSlur.value;
+            if (!sp->isSlur()) {
+                continue;
+            }
+            ChordRest* cr1 = sp->startCR();
+            ChordRest* cr2 = sp->endCR();
```

Comment:
Don't we need a sanity check before using cr1 and cr2? The slur layout code contains many of those.

Reply:
Certainly couldn't hurt! Added.